### PR TITLE
Secrets UI

### PR DESCRIPTION
**Pull request by nikolalsvk**
- [x] Describe manipulating Secrets
- [x] Describe manipulating Environment Variables
- [x] Describe manipulating Configuration Files
- [x] Attaching and detaching a secret to a Team
- [x] Attaching and detaching a secret to a Project
- [x] Mention Environment Variables from a secret in Environment Variable docs
- [x] Mention Configuration Files from a secret in Configuration File docs

Also removed the "custom" word from configuration files, lost the screenshots when describing configuration files and renamed the doc file for configuration files.

### CODE REVIEW

File: source/docs/adding-configuration-files.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+For greater security, the content of your files can be stored as encrypted. We
+strongly recommend that you select this option if you are adding sensitive data
+such as SSH keys. Once created, encrypted file cannot be edited. The identity
+of your file can be determined by the MD5 hash.
+
+### Adding a configuration file from a secret
+
+In order to share the common build configuration, you can add a configuration
+file from a secret that you've previously added. Secrets are more convenient solution
```
Comment (mimimalizam) [inline source/docs/adding-configuration-files.md]: 🕵️  add & added in the same sentence - what do you think of
`In order to share the common build configuration, you can add a configuration file from a secret that you've previously created.`

File: source/docs/adding-configuration-files.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+Content of the configuration file can be entered in the text area belowt the
+file path.
+
+### Encrypting content of configuration files
+
+For greater security, the content of your files can be stored as encrypted. We
+strongly recommend that you select this option if you are adding sensitive data
+such as SSH keys. Once created, encrypted file cannot be edited. The identity
+of your file can be determined by the MD5 hash.
```
Comment (mimimalizam) [inline source/docs/adding-configuration-files.md]: great reference as well - tnx for pointing it out 🙇

File: source/docs/exporting-environment-variables.md [hunk 3]
```diff
@@ -4,15 +4,22 @@ title: Exporting environment variables
...
+the build and deploy process.
 
 ### Adding a new environment variable
 
-To get started, open your project settings, "Environment Variables" tab. Click on "Add an environment variable" to open a form.
+To get started, open your project settings, "Environment Variables" tab. Click
+on "Add an environment variable" to open a form.
 
-For each variable you need to specify its' name and value (content). For instance, you would enter "SECRET_TOKEN" as name and "123abcde" as content.
+For each variable you need to specify its' name and value (content). For
```
Comment (mimimalizam) [inline source/docs/exporting-environment-variables.md]: I believe it should be `its` instead of `its'`

File: source/docs/secrets.md [hunk 4]
```diff
@@ -41,16 +41,171 @@ configuration to their teams to allow non-admin users to use it on their project
...
-To use the entries from the secret in your projects, you need to
-attach the secret to your project, and select the environment
-variables you want to use from the secret.
+To use the entries from the secret in your projects, you need to attach the
+secret to your project. Once attached, all entries will be included in your
+project.
+
+## Setting up a secret via UI
+
+In order to add a secret to your organization, you have to an owner of the
```
Comment (mimimalizam) [inline source/docs/secrets.md]: `you have to an owner` -> `you have to be an owner`

I'd like to suggest

It is necessary to be a member of the organization owners team or a team with the "admin" [permission level](/docs/organizations/permission-levels-in-an-organization.html) in order to create and manage secrets.

On a similar note, we could add 
`Manage secrets` in the table for Owners and Admins [here](https://semaphoreci.com/docs/organizations/permission-levels-in-an-organization.html) as well.

File: source/docs/adding-configuration-files.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+To get started, open your project settings, "Configuration Files" tab. Click to
+add a new file.
+
+For a new file you need to specify a target file path. For instance, if you
+have a project "semaphore" and you want to add a new file `s3.yml` to your
+project's `config` directory, your file path should look like this
+`semaphore/config/s3.yml`.
+
+Content of the configuration file can be entered in the text area belowt the
```
Comment (vonum) [inline source/docs/adding-configuration-files.md]: Typo: `below`

File: source/docs/secrets.md (hunk 4 above)
Comment (vonum) [inline source/docs/secrets.md]: You have to be an

File: source/docs/secrets.md [hunk 6]
```diff
@@ -41,16 +41,171 @@ configuration to their teams to allow non-admin users to use it on their project
...
+In order to attach a secret to a project, click "Edit" next to
+"Projects sharing this Secret" title.
+
+Once "Edit" is clicked, you should see a list of projects that belong to your
+organization. In order to attach a secret to a project, you need to check a box
+next to the project name. You will see a short 'Saved.' message once the secret
+is attached.
+
+To detach the secret from a project, you will need to uncheck the box next to
+the project name. There will be a 'Saved.' message once the secret is detached.
```
Comment (vonum) [inline source/docs/secrets.md]: You will see a "Saved." message...

**Pull request merged**